packages/chart/src/components/LinearChart.tsx

```diff
@@ -114,12 +115,73 @@ const LinearChart = forwardRef<SVGAElement, LinearChartProps>(({ parentHeight, p
     freezeOnceVisible: false
   })
 
+  // Pre render functions
+  const countNumOfTicks = (axis, max) => {
+    let { numTicks } = runtime[axis]
+    if (runtime[axis].viewportNumTicks && runtime[axis].viewportNumTicks[currentViewport]) {
+      numTicks = runtime[axis].viewportNumTicks[currentViewport]
+    }
+    let tickCount = undefined
+
+    if (axis === 'yAxis') {
+      tickCount =
+        isHorizontal && !numTicks
+          ? data.length
+          : isHorizontal && numTicks
+          ? numTicks
+          : !isHorizontal && !numTicks
+          ? undefined
+          : !isHorizontal && numTicks && numTicks
+      // to fix edge case of small numbers with decimals
+      if (tickCount === undefined && !config.dataFormat.roundTo) {
+        // then it is set to Auto
+        if (Number(max) <= 3) {
+          tickCount = 2
+        } else {
+          tickCount = 4 // same default as standalone components
+        }
+      }
+      if (Number(tickCount) > Number(max) && !isHorizontal) {
+        // cap it and round it so its an integer
+        tickCount = Number(min) < 0 ? Math.round(max) * 2 : Math.round(max)
+      }
+    }
+
+    if (axis === 'xAxis') {
+      tickCount =
+        isHorizontal && !numTicks
+          ? undefined
+          : isHorizontal && numTicks
+          ? numTicks
+          : !isHorizontal && !numTicks
+          ? undefined
+          : !isHorizontal && numTicks && numTicks
+      if (isHorizontal && tickCount === undefined && !config.dataFormat.roundTo) {
+        // then it is set to Auto
+        // - check for small numbers situation
+        if (max <= 3) {
+          tickCount = 2
+        } else {
+          tickCount = 4 // same default as standalone components
+        }
+      }
+
+      if (config.visualizationType === 'Forest Plot') {
+        tickCount = config.yAxis.numTicks !== '' ? config.yAxis.numTicks : 4
+      }
+    }
+
+    return tickCount
+  }
+
   // VARS/MEMOS
   const shouldAbbreviate = true
   const isHorizontal = orientation === 'horizontal' || config.visualizationType === 'Forest Plot'
   const isLogarithmicAxis = config.yAxis.type === 'logarithmic'
   const isForestPlot = visualizationType === 'Forest Plot'
   const suffixHasNoSpace = !suffix.includes(' ')
+  const padding = orientation === 'horizontal' ? Number(config.xAxis.size) : Number(config.yAxis.size)
+  const fontSize = { small: 16, medium: 18, large: 20 }
```

Comment:
Can we remove this and use `packages/core/helpers/cove/fontSettings.ts`. I think this is one of those repeated values we need to remove and reducing the amount of repetition or removing it altogether in favor of CSS (if possible) would be helpful.

Reply:
@adamdoe is ok if we keep this here and save the refactor for a more relevant change? Its not really related to work i did - i just moved it up from a lower part of the component in an attempt to keep the variables together

Reply:
Sure, thats fine!

Reply:
@adamdoe Oh sorry I didn't realize how easy the request was! adding in the change 🫡